document super reactions (#6056)
hammerandchisel/discord-api-docs: docs/resources/Channel.md

  "reactions": [
    {
      "count": 1,
      "count_details": {
         "burst": 0,
         "normal": 1
      },
      "me": false,

>       "me_burst": false,

Reviewer: ```suggestion
```

  "reactions": [
    {
      "count": 1,
      "count_details": {
         "burst": 0,
         "normal": 1
      },

>       "me": false,
>       "emoji": {

Reviewer 2: ```suggestion
      "me": false,
      "me_burst": false,
      "emoji": {
```

  "reactions": [
    {
      "count": 1,
      "count_details": {
         "burst": 0,
         "normal": 1
      },
      "me": false,

>       "me_burst": false,

Reviewer: ```suggestion
```

###### Reaction Structure

> | Field         | Type                                                       | Description                                                                                |
> | ------------- | ---------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
> | count         | integer                                                    | times this emoji has been used to react                                                    |
> | count_details | object                                                     | the [reaction count details object](#DOCS_RESOURCES_CHANNEL/reaction-count-details-object) |
> | me            | boolean                                                    | whether the current user reacted using this emoji                                          |
> | emoji         | partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | emoji information                                                                          |
> | burst_colors  | array                                                      | HEX colors used for super reaction                                                         |
> 

Reviewer 2: ```suggestion
| Field         | Type                                                       | Description                                                                            |
| ------------- | ---------------------------------------------------------- | -------------------------------------------------------------------------------------- |
| count         | integer                                                    | Total number of times this emoji has been used to react (including super reacts)       |
| count_details | object                                                     | [Reaction count details object](#DOCS_RESOURCES_CHANNEL/reaction-count-details-object) |
| me            | boolean                                                    | Whether the current user reacted using this emoji                                      |
| me_burst      | boolean                                                    | Whether the current user super-reacted using this emoji                                |
| emoji         | partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | emoji information                                                                      |
| burst_colors  | array                                                      | HEX colors used for super reaction                                                     |

```

Reviewer 2: added `me_burst` and just did some capitalization

### Reaction Count Details Object

> The reaction count details object contains information about super and normal reaction counts.

Reviewer 2: ```suggestion
The reaction count details object contains a breakdown of normal and super reaction counts for the associated emoji.
```

###### Reaction Count Details Structure

| Field  | Type      | Description                   |
| ------ | --------- | ----------------------------- |

> | burst  | integer   | count of super reactions      |

Reviewer 2: ```suggestion
| burst  | integer   | Count of super reactions      |
```

> | normal | integer   | count of normal reactions     |

Reviewer 2: ```suggestion
| normal | integer   | Count of normal reactions     |
```

###### Reaction Structure

| Field         | Type                                                       | Description                                                                                |
| ------------- | ---------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
| count         | integer                                                    | times this emoji has been used to react                                                    |
| count_details | object                                                     | the [reaction count details object](#DOCS_RESOURCES_CHANNEL/reaction-count-details-object) |
| me            | boolean                                                    | whether the current user reacted using this emoji                                          |

> | me_burst      | boolean                                                    | whether the current user reacted with super reaction using this emoji                      |

Reviewer: ```suggestion
```

Create a reaction for the message. This endpoint requires the `READ_MESSAGE_HISTORY` permission to be present on the current user. Additionally, if nobody else has reacted to the message using this emoji, this endpoint requires the `ADD_REACTIONS` permission to be present on the current user. Returns a 204 empty response on success. Fires a [Message Reaction Add](#DOCS_TOPICS_GATEWAY_EVENTS/message-reaction-add) Gateway event.
The `emoji` must be [URL Encoded](https://en.wikipedia.org/wiki/Percent-encoding) or the request will fail with `10014: Unknown Emoji`. To use custom emoji, you must encode it in the format `name:id` with the emoji name and emoji id.

> ###### Query String Params
> 
> | Field   | Type      | Description                                            |
> | ------- | --------- | ------------------------------------------------------ |
> | burst   | boolean   | whether to establish this reaction as a super reaction |
> 

Reviewer: ```suggestion
```
